src/searchPage/searchPageEvents.ts

```diff
@@ -509,3 +536,20 @@ export interface GeneratedAnswerCitationMeta {
     permanentId: string;
     id: string;
 }
+
+export type GeneratedAnswerFeedbackReason = 'irrelevant' | 'notAccurate' | 'outOfDate' | 'harmful' | 'other';
+
+export type GeneratedAnswerRephraseStrategy = 'stepByStep' | 'bulletPoints' | 'summarize';
+
+export interface GeneratedAnswerCitationHoverMeta extends GeneratedAnswerCitationMeta {
+    citationHoverTime: number;
+}
+
+export interface GeneratedAnswerRephraseMeta extends GeneratedAnswerBaseMeta {
+    rephraseStrategy: GeneratedAnswerRephraseStrategy;
```

Comment:
```suggestion
    generatedAnswerFormat: GeneratedAnswerRephraseFormat;
```
Seems to be format in the google doc